### PR TITLE
Vertical stacked bar chart new component tests

### PR DESCRIPTION
**Pull request by v-baambati**

### CODE REVIEW

**Comment by v-baambati** [comment]
Duplicate pull request

**Comment by AtishayMsft** [comment]
@v-baambati Could you resolve the merge conflict in this PR.

**Comment by AtishayMsft** [comment]
> @v-baambati Could you resolve the merge conflict in this PR.

There seem to be new merge conflicts in this PR now.

**Comment by v-baambati** [comment]
Hi @AtishayMsft resolved merge conflicts.

File: packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx [hunk 1]
```diff
@@ -7,9 +7,9 @@ import { IVSChartDataPoint, IVerticalStackedChartProps } from '../../index';
 import { VerticalStackedBarChart } from './VerticalStackedBarChart';
 import { getByClass, getById, testWithWait, testWithoutWait } from '../../utilities/TestUtility.test';
 import { VerticalStackedBarChartBase } from './VerticalStackedBarChart.base';
+import * as utils from '@fluentui/react/lib/Utilities';
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx:10]: why is this entry needed

File: packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx [hunk 2]
```diff
@@ -296,6 +315,62 @@ describe('Vertical stacked bar chart - Subcomponent Legends', () => {
...
+
+  testWithWait(
+    'Should call the handler on mouse leave from line legend',
+    VerticalStackedBarChart,
+    { data: simplePointsWithLine, calloutProps: { doNotLayer: true } },
+    container => {
+      // eslint-disable-next-line
+      const handleMouseLeave = jest.spyOn(VerticalStackedBarChartBase.prototype as any, '_onLegendLeave');
+      const legends = screen.getAllByText((content, element) => element!.tagName.toLowerCase() === 'button');
+      fireEvent.mouseLeave(legends[0]);
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx:341]: I dont see any difference between this test and the previous test

File: packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx (hunk 1 above)
Comment (v-baambati) [reply packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx:10]: Yes Atishay we are mocking getRTL function from utilities class.

File: packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx (hunk 2 above)
Comment (v-baambati) [reply packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx:341]: removed duplicate test case

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/5cb51c1747c839ac8c50eb2c3b527200607e5253'>5cb51c1747c839ac8c50eb2c3b527200607e5253</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by AtishayMsft** [comment]
@v-baambati Could you merge master for this PR.

**Comment by v-baambati** [comment]
Completed @AtishayMsft

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**